sdk/monitor/monitor-ingestion/samples-dev/logsIngestionClient.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { LogsIngestionClient } from "../src/logsIngestionClient";
+import * as dotenv from "dotenv";
+import { DefaultAzureCredential } from "@azure/identity";
+
+dotenv.config();
+
+export async function main() {
+  const logsIngestionEndpoint = process.env.LOGS_INGESTION_ENDPOINT || "logs_ingestion_endpoint";
+  const dcrId = process.env.IMMUTABLE_ID || "immutable_dcr_id";
```

Comment:
process.env.IMMUTABLE_DATA_COLLECTION_RULE_ID better?

Reply:
switched to process.env.DATA_COLLECTION_RULE_ID